boss.conf

```
@@ -1,6 +1,9 @@
 [boss]
-db_type = Fs
+# To use the old fs_storage
 db_path = /var/spool/boss/boss_ruote_db
+# Use the new sequel store
+# if db is specified it takes priority
```

I think this might cause confusion. If both are defined it should probably throw an error.
Or there could be just one option db, which takes file:// or postgres:// url. Not sure if e really need to care about the backwards compatibility for old configs.